src/main/java/net/minecraftforge/client/event/RegisterEntitySpectatorShadersEvent.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) Forge Development LLC and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.minecraftforge.client.event;
+
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.world.entity.EntityType;
+import net.minecraftforge.eventbus.api.Cancelable;
+import net.minecraftforge.eventbus.api.Event;
+import net.minecraftforge.fml.LogicalSide;
+import net.minecraftforge.fml.event.IModBusEvent;
+import net.minecraftforge.fml.javafmlmod.FMLJavaModLoadingContext;
+import org.jetbrains.annotations.ApiStatus;
+
+import java.util.Map;
+
+/**
+ * Allows users to register custom shaders to be used when the player spectates a certain kind of entity.
+ * Vanilla examples of this are the green effect for creepers and the invert effect for endermen.
```

Comment:
The same thing that happened with the previous registration method. It's just been moved to a standalone registration event instead of being a static register method in some class.